src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -41,6 +41,7 @@ public class RevisionsStorage
         private static readonly Slice RevisionsTombstonesSlice;
         private static readonly Slice RevisionsPrefix;
         public static Slice ResolvedFlagByEtagSlice;
+        public long SizeLimit = 32 * 1_024 * 1_024;
```

Comment:
this suggests that we by default should use different thresholds for x64 and x86
1. Rename to `SizeLimitInBytes`
2. `public static long SizeLimitInBytes = (new Size(PlatformDetails.Is32Bits == false ? 32 : 2, SizeUnit.Megabytes)).GetValue(SizeUnit.Bytes);` or sth like that?

Reply:
I suggest not to make is static though

Reply:
we can mark it as readonly? the Q is, if that test needs to change it or not when we will set it to a more reasonable value + tests should use then this value?

Reply:
In the 32bit tests I would like to avoid of having even 2MB large documents.
So I would prefer that the test would be able to set a custom value (like 32KB in this case)

Reply:
sure, so local, but still for 32 bit it should be much smaller

Reply:
agree

Reply:
Done